src/components/Weather.js

```diff
@@ -4,31 +4,58 @@
  * @desc
  */
 import React, { Component } from 'react';
-import DropdownButton from 'react-bootstrap/DropdownButton';
-import Dropdown from 'react-bootstrap/Dropdown';
+import axios from 'axios';
+import CitySelector from './CitySelector';
 
 class Weather extends Component {
+    
+    constructor(props) {
+        super(props);
+        this.state = {
+            city: null,
+            weatherData: [],
+        };
+    }
+
+    callBackendAPI = async (e) => {
+        const weatherQuery = 'http://localhost:5000/weather?city=' + e;
```

Comment:
Just for consistency, make sure this matches the other places that it's being used! Your server port is using 5001 :)